addons/website_event_track/models/event_track.py

```diff
@@ -630,3 +648,91 @@ def _get_track_suggestions(self, restrict_domain=None, limit=None):
         )
 
         return track_candidates[:limit]
+
+    def _get_reminder_times_warning(self):
+        return Markup('<strong><u>%(warning_title)s</u></strong>: %(warning_content)s') % {
+            'warning_title': _('Note'),
+            'warning_content': _(
+                'The start and end times of the talk were not specified when you asked to add them to your calendar, '
+                'therefore the times indicated in this reminder correspond to those of the event.'
+            )
+        } if not self.date else ''
+
+    def _get_external_description(self):
```

Comment:
:socks: in doubt, order methods alphabetically in a given section (aka in this TOOLS sub-section of the file)